Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -166,9 +167,30 @@ private void Initialize()
             #endregion Managers Registration
 
             #region SDK Initialization
-
-            // TODO Microsoft.MixedReality.Toolkit - SDK Initialization
-            AddManager(typeof(IMixedRealityDeviceManager), new WindowsMixedRealityDeviceManager("Mixed Reality Device manager", 10));
+            switch (Application.platform)
+            {
+                case RuntimePlatform.WindowsPlayer:
+                case RuntimePlatform.WindowsEditor:
+                    AddManager(typeof(IMixedRealityDeviceManager), new OpenVRDeviceManager("OpenVR Device Manager", 10));
```

Comment:
What if I wanted to use an open vr controller for wsa?

Reply:
Now that is a subject for later.  Not even going to ponder that until my test hardware arrives :D

Reply:
I'm also wondering if the WMR controllers show up in the openvr APIs

Reply:
That I can answer, no.  Or at least I don't think so.
The non WSA XR stuff only "lights up" if the VR platform is set to "OpenVR".
In UWP it's set to "Windows Mixed Reality".  But it doesn't hurt, to test, LATER!!

Reply:
So then in reality we shouldn't be checking the build platform, but which SDK is currently loaded in?

Reply:
Temporary measure while I add the OpenVR platform.  Will review SDK detection in a separate PR, as that is not the focus of this PR :D

Reply:
> I'm also wondering if the WMR controllers show up in the openvr APIs

They might show up if you're using the [WMR/Steam utility](https://store.steampowered.com/app/719950/Windows_Mixed_Reality_for_SteamVR/).

Reply:
That's how I'm testing this at the moment :D 
Using the SteamVR wrapper in Unity standalone with WMR.  I have other test kit "on the way" to test everything with the other SDK's